oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -1169,7 +1212,12 @@ impl Runtime {
             config.node_subname(entrypoint),
             new_node_id.0
         );
-        self.node_configure_instance(new_node_id, &new_node_name, label);
+        self.node_configure_instance(
+            new_node_id,
+            &new_node_name,
+            label,
+            &NodePrivilege::default(),
```

Comment:
Yes, but the privilege will be granted by the runtime itself (e.g. based on the Wasm code hash), not by the creator node. Does this make sense? Let me know if you think we are not on the same page about this, because I suspect this is not really "standard" in other IFC system.

Reply:
I don't think I fully follow the way you are doing it, but I think I don't need to fully follow how privilege movement works right now before you merge this.